hieradata/org/lsst/role/rke.yaml

```diff
@@ -1,8 +1,35 @@
 ---
 classes:
   - "profile::core::common"
+  - "profile::core::kernel"
+  - "profile::core::kernel::pquota"
+  - "profile::core::selinux"
+  - "profile::app::helm"
+  - "profile::app::rke"
+  - "profile::app::macvlan"
   - "docker"
   - "sysctl::values"
+  - "yum"
+packages:
```

Comment:
Declaring large numbers of resources in hiera is an anti-pattern. It's convenient for small things, but implementing core functionality for a module breaks encapsulation, prevents reuse, and causes our hierarchy to sprawl. More importantly, this makes it hard for us to reason about where resources are defined. Right now resources are defined from Puppet - _unless they're coming from hiera_ - is confusing if you're not deeply experienced. 

Because we're under pressure to get RKE delivered I won't block this pull request, so let's refactor this in a followup PR.